### PR TITLE
feat: these for several path

### PR DESCRIPTION
**Pull request by Mehrbod2002**
Fixes : #70 
Allow several paths with `these`

### CODE REVIEW

**Review by Byron — CHANGES_REQUESTED** [review]
The idea of `these` would be to pass multiple paths to `commands()` at once. What makes this hard isn't the API though, it's validating that it even makes sense for the launchers that are offered on each platform. Not all of them may support multiple paths at all, which is when one would have to let it fall back to launching each command individually.

Besides that, any use of `unsafe` is unacceptable  here and also not required. Instead of `PathIter`, try to use something like `impl IntoIterator<Item = impl AsRef<OsStr>>` or the version with explicit type constraints using `where I: IntoIterator<Item = R>, R: AsRef<OsStr>`, you will probably have to fiddle with lifetimes a bit.

**Comment by Mehrbod2002** [comment]
@Byron 
To be sure , multi path means open a browser with different path on different tabs ?

**Comment by Byron** [comment]
I think it means that launchers who can take multiple paths at once can be provided with multiple paths. The effect of this is defined by its implementation.

**Comment by Mehrbod2002** [comment]
@Byron 
I think there is a misunderstanding about this impl . the command with several means open an browser with multi path in multi tabs ?

File: src/lib.rs [hunk 1]
```diff
@@ -152,6 +153,22 @@ pub fn that(path: impl AsRef<OsStr>) -> io::Result<()> {
     Err(last_err.expect("no launcher worked, at least one error"))
 }
 
+pub fn these(
+    path: impl IntoIterator<Item = impl AsRef<OsStr>> + std::convert::AsRef<std::ffi::OsStr>,
```
Comment (Byron) [inline src/lib.rs:157]: There is no need for any function that can be emulated with `for path in paths { open::that(path)?; }` Instead, in order to implement it as described in #70, launchers must be fed multiple paths if they support it.

**Review by Byron — COMMENTED** [review]
Thanks for trying to contribute!

However, it seems there is difficulty in communication and/or understanding, which is why I would prefer abandoning this PR instead of trying to push it into shape. My time is too scarce to be able to engage here.

Thanks for your understanding.

**Pull request closed without merging**